sdk/scheduler/src/main/java/com/mesosphere/sdk/state/StateStoreUtils.java

```diff
@@ -33,7 +39,7 @@
     private static final String UNINSTALLING_PROPERTY_KEY = "uninstalling";
     private static final String LAST_COMPLETED_UPDATE_TYPE_KEY = "last-completed-update-type";
     private static final String PROPERTY_TASK_INFO_SUFFIX = ":task-status";
-    private static final int MAX_VALUE_LENGTH_BYTES = 1024 * 1024; // 1MB
+    protected static final int MAX_VALUE_LENGTH_BYTES = 1024 * 1024; // 1MB
```

Comment:
Yes, that was the first thing I tried, but then I get the following:
```
Error:(79, 53) java: MAX_VALUE_LENGTH_BYTES has private access in com.mesosphere.sdk.state.StateStoreUtils
```
in my unit tests. Is there some limitation where one can only annotate a non-primitive data type in Java?